pkg/features/kube_features.go

```diff
@@ -934,6 +934,19 @@ const (
 	//
 	// Enables In-Place Pod Vertical Scaling
 	InPlacePodVerticalScaling featuregate.Feature = "InPlacePodVerticalScaling"
+
+	// owner: @Sh4d1,@RyanAoh
+	// kep: http://kep.k8s.io/1860
+	// alpha: v1.29
+	// LoadBalancerIPMode enables the IPMode field in the LoadBalancerIngress status of a Service
+	LoadBalancerIPMode featuregate.Feature = "LoadBalancerIPMode"
+
+	// owner: @AxeZhan
```

Comment:
I was told that these features should really be in alphabetical order.  All arent but that is the goal.

Reply:
Well, I thought this was placed in time order(😂 new one placed after the old one). I'll place this one in alphabetical order.